packages/xo-server-sdn-controller/src/index.js

```diff
@@ -564,7 +564,7 @@ class SDNController extends EventEmitter {
 
           // Re-elect a center to apply the VNI
           const privateNetwork = this._privateNetworks[
-            network.other_config['private-network-uuid']
+            network.other_config['xo:sdn-controller:private-network-uuid']
```

Comment:
I think we should split chore and fixes.

Reply:
I can do it yes.

Reply:
I removed the 2 fix from the commits, all the other are does not change the behavior of the code.